tools/autograd/gen_variable_type.py

```diff
@@ -395,17 +413,26 @@ def emit_save_outputs():
             return CONDITIONAL.substitute(cond='grad_fn', statements=stmts)
         return ''
 
+    def emit_check_inplace():
```

Comment:
it's a little strange that in the rest of the file 'inplace' is non-overlapping with is_out_fn but here they mean the same thing.  Is there a name that encompasses both?

Reply:
I'm open to suggestions

Reply:
overwrite?

Reply:
oh, later you use "modifies_arguments" -- I like that more.